main.tf

```diff
@@ -222,18 +220,27 @@ locals {
   ingress_from_cidr_range_override_defined = length(var.security_group_override_ingress_from_cidr_map) > 1 ? true : false
 
   ingress_existing_sg_defined = length(var.security_group_ingress_with_existing_security_groups_map) > 1 ? true : false
+
+  security_group_name            = var.security_group_name != null ? var.security_group_name : local.common_name
+  security_group_name_prefix     = coalesce(var.security_group_name_prefix, var.common_prefix, "anyscale-security-group-")
+  amp_security_group_name        = var.anyscale_machine_pool_security_group_name != null ? var.anyscale_machine_pool_security_group_name : local.common_name != null ? "${local.common_name}-machine-pool-sg" : null
+  amp_security_group_name_prefix = coalesce(var.anyscale_machine_pool_security_group_name_prefix, var.common_prefix, "anyscale-machine-pool-sg-")
 }
 
 module "aws_anyscale_securitygroup_self" {
   source = "./modules/aws-anyscale-securitygroups"
   tags   = local.securitygroup_tags
   vpc_id = coalesce(var.existing_vpc_id, module.aws_anyscale_vpc.vpc_id)
 
-  security_group_name                       = local.security_group_name
-  security_group_name_prefix                = local.security_group_name_prefix
-  create_anyscale_public_ingress            = var.security_group_create_anyscale_public_ingress
+  security_group_name        = local.security_group_name
+  security_group_name_prefix = local.security_group_name_prefix
+  # create_anyscale_public_ingress            = var.security_group_create_anyscale_public_ingress
```

Comment:
Add a comment to say this is for v1 stack not needed?
Or just delete them all together?